subcommittees/mentoring/ITProposal.md

```diff
@@ -0,0 +1,114 @@
+  curriculum of [Software][swc-core] or Data Carpentry.
+- Instructor in Training: can teach and organize a mini-lesson,
+  session, or workshop under the supervision of a badged Software or
+  Data Carpentry instructor.
+- Instructor: can lead, teach, and organize a mini-lesson, session, or
+  workshop.
+- Program Coordinator: TODO (is this the Executive Director?)
+
+## Badges
+
+Anyone interested in joining the Software and/or Data Carpentry team
+of instructors **must** complete the Instructor Training in any format
+offered by Software and/or Data Carpentry.
+
+Software and Data Carpentry will assign "Instructor in Training"
+badges to people who have successfully completed a [checkout
+session](#checkout-session).  After badging, Instructors in Training
+have one year to teach one mini-lesson, one session, or one workshop
+under the supervision of a badged Software or Data Carpentry
+Instructor.  After each mini-lesson, session, or workshop, the
+supervising Instructor **can** recommend the Instructor in Training
+for the Instructor badge.  The Software and/or Data Carpentry
+Executive Director will assign Instructor badges following the
+recommendations of the supervising Instructor.  The Instructor in
+Training badge expires after a year, by which time the Instructor in
+Training should have become a badged Instructor.  If the Instructor in
+Training has not become and Instructor and still wants to become one,
+they must re-enter the Instructor Training process at the beginning.
+
+## Checkout Session
+
+1.  Watch a 5min lesson video of a Software or Data Carpentry instructor
+    about one of the lessons on the currículum.
+
+2.  Sign up for one checkout session.
+
+3.  Join the checkout session.
+
+4.  Deliver a 5min lesson about one of the lessons on the curriculum.
+
+5.  Receive feedbacks about the lesson.
+
+6.  Watch people at the checkout session.
+
+7.  Provide feedback on the lessons delivered by others.
+
+## Mentoring
+
+Instructors in Training can wait for a invitation from the Program
+Coordinator to join one workshop or can request the Mentoring
+Subcommittee assistance to organize one mini-lesson, session, or
+workshop.
+
+### Invitation from the Program Coordinator
+
+Timeframe: at least one month after invitation from the Program
+Coordinator.  TODO: is this the timeframe from final-project →
+invitation, or from invitation → workshop?
+
+1. The Program Coordinator invites the Instructor in Training to
+   co-teach at a workshop that already has an Instructor.
+
+2. The supervising Instructor will help the Instructor in Training
+   review the lesson s/he will deliver at the workshop.
+
+3. The supervising Instructor and Instructor in Training **must** meet
+   online 1 and 3 weeks before the workshop.
+
+4. After the workshop, the supervising Instructor **may** recommend
+   the Instructor in Training for the Instructor badge.
+
+### Request Mentoring Subcommittee assistance
```

Comment:
This is somewhat inconsistent with what the *Mentoring* section says. The language needs to be clearer.
In *Mentoring* "an request the Mentoring Subcommittee assistance to organize one mini-lesson, session, or
workshop." but then the details of the assistance mention only workshops.